src/repositories/media.repository.ts

```diff
@@ -12,11 +12,7 @@ import Chance = require('chance');
 import { MediaRepositoryCommentResponseRootObject } from '../responses/media.repository.configure.response';
 
 export class MediaRepository extends Repository {
-  public async info({
-    mediaId,
-  }: {
-    mediaId: string;
-  }) {
+  public async info(mediaId: string) {
     const { body } = await this.client.request.send({
```

Comment:
It is not enough to just generate response interface, you need to link it in generic and function return signature, like here
https://github.com/dilame/instagram-private-api/blob/master/src/repositories/media.repository.ts#L100

Reply:
👌